AstroPhotometry/View/FileTree.xaml.cs

```diff
@@ -56,8 +62,7 @@ private void InitializeFileSystemObjects()
                 });
 
             // start in specific folder 
-            // TODO: make it start from desktop (one layer in)
-            PreSelect(Environment.GetFolderPath(Environment.SpecialFolder.Desktop));
+            //PreSelect(Environment.GetFolderPath(Environment.SpecialFolder.Desktop));
```

Comment:
Necessary?

Reply:
No, because i added desktop to be on top, so i deleted the opening of the folder by path.